cmake/IncludeWhatYouUse.cmake

```diff
@@ -0,0 +1,11 @@
+if(ENABLE_IncludeWhatYouUse)
+  find_program(IncludeWhatYouUse include-what-you-use)
```

Comment:
`IncludeWhatYouUse_FOUND`

Reply:
Why not to turn on iwyu when find the program automatically? i think this option is not necessary.

Reply:
because it run very slow,  Just let developer run it  .